packages/wallets/src/components/WalletCard/WalletCard.scss

```diff
@@ -0,0 +1,50 @@
+.wallets-card {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    &__data {
+        width: 24rem;
+        height: 14.4rem;
+        border-radius: 8px;
+        background-color: rgb(242, 200, 200);
+        position: relative;
+
+        &__details {
+            display: flex;
+            width: 100%;
+            height: 100%;
+            padding: 1.6rem;
+            flex-direction: column;
+            justify-content: space-between;
+            align-items: flex-start;
+
+            &-icon {
+                width: 3.2rem;
+                height: 3.2rem;
+            }
+
+            &-balance {
+                display: flex;
+                flex-direction: column;
+                align-items: flex-start;
+                gap: 0.4rem;
+
+                & > h3 {
```

Comment:
yo can we not use h3 here? doesn't seem semantically correct. you could instead use different classnames